package.json

```diff
@@ -2,17 +2,18 @@
   "name": "react-colorful",
   "version": "2.3.1",
   "description": "A tiny color picker component for modern React apps",
-  "source": "src/hex.tsx",
+  "source": "src/index.ts",
   "main": "dist/index.js",
   "module": "dist/index.module.js",
   "esmodule": "dist/index.esmodule.js",
   "umd:main": "dist/index.umd.js",
-  "types": "dist/hex.d.ts",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
   "scripts": {
     "lint": "eslint src/**/*.{ts,tsx} demo/src/**/*.js",
     "size": "npm run build && size-limit",
     "test": "jest tests",
-    "build": "node ./build-packages.js",
+    "build": "microbundle build --entry src/index.ts --jsx React.createElement --name react-colorful --css-modules true --sourcemap false --tsconfig tsconfig.build.json",
```

Comment:
Do we really need to keep sourcemaps disabled? I was against it back when you did that, but figured I'd hold off as it was a fair bit of junk at the time.

But now there's just a few. Sourcemaps are quite useful when debugging.

Reply:
With minified bundles I think sourcemaps are sort of required.

Reply:
Plus it's only weight in a dev environment, which will likely be 1000x the size as that's just the state of node. Losing a few kb from `node-modules` won't go noticed.

Reply:
We dropped them off because the total size of NPM package was huge. Probably it's no longer a problem since we publish only one dist folder instead of 7.

Reply:
I got them back. The NPM package will be:
```
unpacked size: 156.9 kB
total files:   44
```